src/main_nep/fitness.cu

```diff
@@ -162,11 +162,7 @@ void Fitness::compute(
         para,
         dummy_solution.data(),
         train_set[n],
-#ifdef USE_FIXED_SCALER
-        false,
-#else
-        true,
-#endif
```

Comment:
Why drop the fixed scaler? What was it used for before?

Reply:
To perfectly restart a training, the descriptor normolizaiton should not be changed; otherwise the descriptors might change dramatically if the dataset has changed a lot, leadding to abrupt change of optimization direction. The macro `USE_FIXED_SCLAER` was used to enforce unchanged descriptor normalization, but it was used by myself only beefore (for training the foundation model). Now it is used for restarting from the foundation model.

Reply:
Okay, makes sense!